Intake/Onboarding/InterestingModules.swift

```diff
@@ -16,24 +16,21 @@ struct InterestingModules: View {
     
     var body: some View {
```

Comment:
Great work on updating the texts in the onboarding flow!

Please be sure that you use localized strings from the Strings catalog. You can learn more about localization here and I would highly recommend watching the video to completely understand the localization process that we use in Spezi-based applications: https://developer.apple.com/wwdc23/10155

What's important is that you actually build the application once on your Mac (via the Simulator) and not just use the SwiftUI previews. Otherwise, the changes in code won't update the String catalog.
The file of the String catalog can be found here: https://github.com/CS342/2024-Intake/blob/main/Intake/Resources/Localizable.xcstrings